packages/neuron-ui/src/widgets/Button/index.tsx

```diff
@@ -14,7 +14,7 @@ const Button = ({
   children,
   ...rest
 }: {
-  type?: 'default' | 'cancel' | 'ok' | 'submit' | 'confirm' | 'primary' | 'reset'
+  type?: 'default' | 'cancel' | 'ok' | 'submit' | 'confirm' | 'primary' | 'reset' | 'text'
```

Comment:
What's the difference between `default` and `text`

Reply:
Default has border, `text` means a button without border.

Reply:
Got it